zerver/lib/soft_deactivation.py

```diff
@@ -191,6 +209,9 @@ def maybe_catch_up_soft_deactivated_user(user_profile):
             event_type='user_soft_activated',
             event_time=timezone_now()
         )
+        logger.info('Soft Reactivated user with (email, id): (%s, %s)' %
+                    (user_profile.email, user_profile.id))
```

Comment:
This logs like this, which is quite verbose; we don't need to explain the format in the log lines:
```
2017-08-16 04:32:58,808 INFO     Soft Reactivated user with (email, id): (hamlet@zulip.com, 30)
```

I would instead have the format be: "Soft reactivated user 30 (hamlet@zulip.com)".